src/TServer.cpp

```diff
@@ -433,24 +438,24 @@ void TServer::InsertClient(const std::shared_ptr<TClient>& NewClient) {
     (void)mClients.insert(NewClient);
 }
 
-void TServer::HandlePosition(TClient& c, const std::string& Packet) {
+bool TServer::HandlePosition(TClient& c, const std::string& Packet) {
```

Comment:
In the future it would be great to do this on the vector<>, not a string packet, since substr() and friends are super unsafe.